typescript/lib/constructor_calls.ts

```diff
@@ -0,0 +1,31 @@
+import * as bindingsJs from '../kurtosis_core_rpc_api_bindings/api_container_service_pb'; //TODO (comment) - I cannont merge with above line unfortunately, can only import one module at a time MAXIMUM
+import { ServiceID } from './services/service';
+
+
+export function newGetExecCommandArgs(serviceId: ServiceID, command: string[]): bindingsJs.ExecCommandArgs {
+    let result = new bindingsJs.ExecCommandArgs(); //TODO (comment) - this was also a pointer
+    result.setServiceId(serviceId);
+    result.setCommandArgsList(command);
+
+    return result;
+}
+
+export function newGetGenerateFilesArgs(serviceId: ServiceID, fileGenerationOpts: Map<string, bindingsJs.FileGenerationOptions>): bindingsJs.GenerateFilesArgs {
+    let result = new bindingsJs.GenerateFilesArgs(); //TODO (comment) this was a pointer
```

Comment:
Here, and everywhere else in this PR - default to using `const` first, and only use `let` if you actually want the mutability. This reduces mutability in the program, which makes for a safer program

Reply:
`result.setServiceId` should work regardless since a const just prevent the variable to be reassigned, it doesn't mean that you can't add to the list of the existing constant. This post explains it very well: https://stackoverflow.com/questions/23436437/why-can-i-change-a-constant-object-in-javascript